install/kubernetes/experimental-install.yaml

```diff
@@ -1181,15 +1181,15 @@ spec:
       serviceAccountName: "hubble-ui"
       containers:
         - name: frontend
-          image: "quay.io/cilium/hubble-ui:latest"
+          image: quay.io/cilium/hubble-ui:latest
```

Comment:
Seems like we lost the quotes here. We should probably add a `quote` to the `"container.image"` teimplate